tools/scripts/platform/linux/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
+    "configurations": [
```

Comment:
As much as I like the idea of being able to debug graphically, I think we shouldn't maintain ourselves config files for third-party IDE's. We do it for XSDK because we use Xilinx SDK. We do it for CrossCore because we use ADUCM etc. but VSCode is just an arbitrary choice that no-OS users might not agree with. On top of this, VSCode is going to evolve and somebody will need to maintain the files in this commit, do we want that ?

Reply:
I think is better than nothing. If some don't like to debug with VSCode, make is still available without it.
It seems that I forgot to add a make run :)

Reply:
Regarding the maintainance, maybe we can write some where it was tested with version x.x.x.

Reply:
It's not better than nothing - we already support 3 IDE's, why should we have a 4th if there is no need ?
There is no standard IDE for the linux platform. In fact, I think most people on that platform prefer vim + plugins.